internal/restinterface/tls/tls.go

```diff
@@ -32,7 +32,8 @@ const (
 
 var (
 	certFilePath atomic.Value
-	handler      PSKHandler
+	// handler      PSKHandler
```

Comment:
any reason for preserving the code in comments

Reply:
@suresh-lc  Thank you for comment. Done!